aat/src/aat/resources/features/F-099 SearchParty (GlobalSearch)/F-099.feature

```diff
@@ -0,0 +1,144 @@
+    And a call [fetch Multiple Pages Case Type Schema to verify comma separated case field names in SearchPartyName column(AC5)] will get the expected response as in [S-099.1_Get_MultiplePagesCaseType].
+    And a call [fetch Conditionals Case Type Schema to verify complex field name in columns SearchPartyName(AC3), SearchPartyEmailAddress(AC7), SearchPartyAddressLine1(AC9), SearchPartyPostCode(AC11), SearchPartyDob(AC13)] will get the expected response as in [S-099.1_Get_ConditionalsCaseType]
+    And a call [fetch Retain Hidden Value Case Type Schema to verify comma separated complex field names in SearchPartyName column(AC4) and comma separated combination of top level field names and complex field names in SearchPartyName column(AC5)] will get the expected response as in [S-099.1_Get_RetainHiddenValueCaseType]
+    And a call [fetch Tabs Case Type Schema to verify blank value in the columns SearchPartyName(AC25), SearchPartyEmailAddress(AC26), SearchPartyAddressLine1(AC27), SearchPartyDoB(AC28) and SearchPartyPostCode(AC29)] will get the expected response as in [S-099.1_Get_TabsCaseType]
+
+  @S-099.2 @AC-14
+  Scenario: Import Definition file with in-correct value in "CaseTypeId" column of the SearchParty tab
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains an invalid value (not matching the CaseType tab) in the configured 'CaseTypeId' column of the SearchParty tab]
+    And it is submitted to call the [Import definition file] operation of [CCD Definition Store]
+    Then a negative response is received
+    And the response has all other details as expected
+    And a call [to fetch incorrect caseTypeId schema] will get the expected response as in [S-099.2_Get_InvalidCaseType].
+
+  @S-099.3 @AC-15
+  Scenario: Import Definition file with invalid complex case field name in the "SearchPartyDoB" column of the SearchCriteria tab
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains correctly configured SearchParty tab with valid case field names in the SearchPartyName column]
+    And the request [contains correctly configured SearchParty tab with valid case field names in SearchPartyEmailAddress, SearchPartyAddressLine1 and SearchPartyPostCode columns]
+    And the request [contains invalid complex field name in the SearchPartyDoB column]
+    And it is submitted to call the [Import definition file] operation of [CCD Definition Store]
+    Then a negative response is received
+    And the response has all other details as expected
+    And a call [verify search party data has not been updated] will get the expected response as in [F-099_VerifyComplexFieldsOrderingSnapshotUnchanged].
+
+  @S-099.4 @AC-16
+  Scenario: Import Definition file with invalid top level case field name in the "SearchPartyDoB" column of the SearchCriteria tab
```

Comment:
Did you mean search criteria or search party tab?

Reply:
Copy and pasted from the JIRA ticket - where they are also incorrect ! So will fix in both places

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/1059/commits/d41404b1f9cf0243dc133d9bf075292529c11e74

Reply:
Fixed in d41404b